content/posts/2021-11-18-relicensing-update.md

```diff
@@ -0,0 +1,328 @@
+  <text x="50"  y="115" style="fill:gray;font-size: 12px;" text-anchor="middle">2011</text>
+  <text x="100" y="115" style="fill:gray;font-size: 12px;" text-anchor="middle">2012</text>
+  <text x="150" y="115" style="fill:gray;font-size: 12px;" text-anchor="middle">2013</text>
+  <text x="200" y="115" style="fill:gray;font-size: 12px;" text-anchor="middle">2014</text>
+  <text x="250" y="115" style="fill:gray;font-size: 12px;" text-anchor="middle">2015</text>
+  <text x="300" y="115" style="fill:gray;font-size: 12px;" text-anchor="middle">2016</text>
+  <text x="350" y="115" style="fill:gray;font-size: 12px;" text-anchor="middle">2017</text>
+  <text x="400" y="115" style="fill:gray;font-size: 12px;" text-anchor="middle">2018</text>
+  <text x="450" y="115" style="fill:gray;font-size: 12px;" text-anchor="middle">2019</text>
+  <text x="500" y="115" style="fill:gray;font-size: 12px;" text-anchor="middle">2020</text>
+  <text x="550" y="115" style="fill:gray;font-size: 12px;" text-anchor="middle">2021</text>
+  <text x="600" y="115" style="fill:gray;font-size: 12px;" text-anchor="middle">2022</text>
+  <text x="650" y="115" style="fill:gray;font-size: 12px;" text-anchor="middle">2023</text>
+  <defs>
+    <linearGradient id="grad1" x1="0%" y1="0%" x2="100%" y2="0%">
+      <stop offset="0%" style="stop-color:rgb(166, 191, 255);stop-opacity:0" />
+      <stop offset="10%" style="stop-color:rgb(166, 191, 255);stop-opacity:1" />
+      <stop offset="90%" style="stop-color:rgb(166, 191, 255);stop-opacity:1" />
+      <stop offset="100%" style="stop-color:rgb(166, 191, 255);stop-opacity:0" />
+    </linearGradient>
+  </defs>
+  <g class="fragment">
+  <!-- First text block is 2011-2015-ish -->
+  <rect x="75" y="40" width="175" height="50" fill="url(#grad1)"/>
+  <!-- Middle of first text block is 2013 -->
+  <text x="150" y="60" style="fill:black;font-size: 16px;" text-anchor="middle">
+  <tspan x="160" dy="0em">Need for license</tspan>
+  <tspan x="160" dy="1em">change emerges</tspan>
+  </text>
+  </g>
+  <g class="fragment">
+  <rect x="255" y="40" width="100" height="50" fill="url(#grad1)"/>
+  <text x="325" y="60" style="fill:black;font-size: 16px;" text-anchor="middle">
+    <tspan x="310" dy="0em">Deciding new</tspan>
+    <tspan x="310" dy="1em">license</tspan>
+  </text>
+  </g>
+  <g class="fragment">
+  2018-now (2022?)〉 Get all code covered by new license.
+  <rect x="350" y="40" width="275" height="50" fill="url(#grad1)"/>
+  <text x="415" y="60" style="fill:black;font-size: 16px;" text-anchor="middle">
+    <tspan x="490" dy="0em">Get all code covered</tspan>
+    <tspan x="490" dy="1em"> by new license</tspan>
+  </text>
+  </g>
+  <g class="fragment">
+  <rect x="625" y="40" width="85" height="50" fill="url(#grad1)"/>
+  <text x="625" y="60" style="fill:black;font-size: 16px;" text-anchor="middle">
+    <tspan x="670" dy="0em">end</tspan>
+    <tspan x="670" dy="1em">game</tspan>
+  </text>
+  </g>
+</svg>
+</img>
```

Comment:
This isn't rendering correctly for me in Chrome on the mac, at least [this preview](https://github.com/llvm/llvm-blog-www/blob/ec112a141c34819724f073158348ae9d1c124c48/content/posts/2021-11-18-relicensing-update.md) doesn't work.

Reply:
This is again github review view vs the view on the blog post. On the blog post (i.e. the hugo framework processing this markdown vs github processing), this display as expected. However, I did now check the blog output more closely with different browsers (chrome, firefox, safari). On safari the colours shown are strangely different. I'll just go for screenshots to have more consistency across browsers. Thanks!